src/hostedPages/activities/form/TraditionalForm.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { Button, CircularSpinner } from '../../../atoms'
+import classes from './form.module.scss'
+import { Question } from '../../../molecules'
+import { useWizardForm } from '../../../hooks/useWizardForm'
+import { WizardFormProps } from '../../../types'
+import layoutClasses from '../../layouts/HostedPageLayout/hostedPageLayout.module.scss'
+
+export const TraditionalForm = ({
+  form,
+  onSubmit,
+  buttonLabels,
+  evaluateDisplayConditions,
+  errorLabels,
+  questionTypeConfig = {},
+  storedAnswers,
+  onAnswersChange,
+}: WizardFormProps) => {
+  const {
+    submitForm,
+    formMethods: { control, getValues },
+    errors,
+    isEvaluatingQuestionVisibility,
+  } = useWizardForm({
+    questions: form.questions,
+    onSubmit,
+    evaluateDisplayConditions,
+    errorLabels,
+    storedAnswers,
+    onAnswersChange,
+  })
+
+  return (
+    <>
+      <main
+        id="ahp_main_content_with_scroll_hint"
+        className={`${layoutClasses.main_content} traditional_form`}
+        style={{ marginBottom: 0 }}
+      >
+        <div className={`${classes.container}`}>
+          {isEvaluatingQuestionVisibility ? (
+            <div className={classes.loadingContainer}>
+              <CircularSpinner size="sm" />
+            </div>
+          ) : (
+            <div>
+              <div className="flex flex-col space-y-8">
```

Comment:
AFAIK, we don't use Tailwind in the UI library so these classes don't do anything?